task/04-arrays-tasks.js

```diff
@@ -304,7 +335,15 @@ function get3TopItems(arr) {
  *   [ 1, '2' ] => 1
  */
 function getPositivesCount(arr) {
-   throw new Error('Not implemented');
+
```

Comment:
There is no need to always add new line before `return` statement if there are no other logical blocks before it